readme.md

```diff
@@ -27,9 +27,9 @@ npm install to-vfile
 ```js
 import {toVFile} from 'to-vfile'
 
-console.log(toVFile('readme.md'))
-console.log(toVFile.readSync('.git/HEAD'))
-console.log(toVFile.readSync('.git/HEAD', 'utf8'))
+console.log(toVFile(new URL('./readme.md', import.meta.url)))
+console.log(toVFile.readSync(new URL('./.git/HEAD', import.meta.url)))
+console.log(toVFile.readSync(new URL('./.git/HEAD', import.meta.url), 'utf8'))
```

Comment:
Hmm, I think we don’t need to change all of them? One, to show that it works, might be fine?